### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by ChiefDarunia** [ValveSoftware/Dota2-Gameplay#3100] [Q&A]
I made a bug report and I am not sure if its getting resolved or if their is anything else I should do to make sure it gets resolved. My venomancer's aztock helmet is gone and I have a few other skin pieces that have vanished as well. I can't remember them at this moment and I will look through and make a list. 

So the questions are is there more that I should be doing to ensure this problem will be fixed, How long will it take, and is anyone else having this problem.

**Comment by rox008** [comment]
I think you must report the error exclusively to the help desk[.](https://homephix.co.uk/shop/over-the-door-towel-hooks/) It will get resolved immediately.

**Comment by trichome3** [comment]
How long does the administration take too answer one query[.](https://sellerdash.pk/) I have submitted a request regarding an error I have been facing, but I still got no answer. I think the optimal time duration is 48 hours. Is it?

**Comment by kylogenetictree** [comment]
I think it takes 36 to 48 hours for one request to be entertained. I am not sure about Git Community[,](https://sellerdash.pk/) but usually the duration is as I have mentioned it.

**Comment by torain0104** [comment]
i got error 127. please help me
![1](https://user-images.githubusercontent.com/112672794/215304720-557df635-60d7-4142-be26-d4f843404ae8.png)
![2](https://user-images.githubusercontent.com/112672794/215304758-ffcaa043-160b-4435-a493-84c677847529.png)
